app/core/Tracker.coffee

```diff
@@ -137,6 +143,9 @@ module.exports = class Tracker extends CocoClass
       gaFieldObject.eventLabel = properties.label if properties.label?
       gaFieldObject.eventValue = properties.value if properties.value?
 
+      gaFieldObject.dimmension0 = me.getHomePageTestGroup()
+      gaFieldObject.dimmension1 = experiments.getRequestAQuoteGroup(me)
```

Comment:
May not affect anything if this is just a variable we use ourselves, but there's an extra `m` here for `dimension`.

Reply:
+1 on spelling typo.
Also it would be helpful if you can add a comment here explaining what this is for, and how results will be tracked on GA.

Reply:
Also do we want this dimension for every event?
It seems we are tracking event `Header Request Quote CTA` only when `getRequestAQuoteGroup` returns `request-a-quote-header`, but the dimension is being added for every event? Can you explain a bit about what will be tracked on GA and how?

Reply:
Good catch with the typo that is critical for GA reporting.

Re reporting - this does attach to every event that we send.  GA allows us to add twenty custom dimensions to our events.  The dimensions are then exposed in the reporting interface allow events to be segmented along one or more dimensions.  Using the dimensions allows us to keep labels the same while supporting multiple experiments at the same time.

Adding to every event allows us to tag all events with a users experiment group, ensuring that we don't miss any events and giving David to do some ad-hoc analysis as required.